newspaper/extractors.py

```diff
@@ -475,7 +475,10 @@ def get_meta_type(self, doc):
     def get_meta_description(self, doc):
         """If the article has meta description set in the source, use that
         """
-        return self.get_meta_content(doc, "meta[name=description]")
+        if len(self.get_meta_content(doc, "meta[name=description]")) > len(self.get_meta_content(doc, "meta[property=description]")):
```

Comment:
This adds unnecessary cpu cycles to parsing, since we call `self.get_meta_content(doc, "meta[name=description]")` multiple times when we only need to call it once.

How about this:
```
meta_name_text = self.get_meta_content(doc, "meta[name=description]")
meta_property_text = self.get_meta_content(doc, "meta[property=description]")
if len(meta_name_text) > len(meta_property_text):
  return meta_name_text
return meta_property_text
```

Or even shorter:
```
return self.get_meta_content(doc, "meta[name=description]") or self.get_meta_content(doc, "meta[property=description]")
```

Reply:
Also, as a design question, can it ever be the case that the *shorter* meta description is the correct description?